### PR TITLE
feat(agent): docker image

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR adds the dockerfile for the agent
## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: agent/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM kubeshop/tracetest:v0.13.9
```
Comment (schoren) [inline agent/Dockerfile:1]: this looks like a very easy thing to forget to update on every release. can we make it dynamic somehow?

Comment (mathnogueira) [reply agent/Dockerfile:1]: do you think it's worth adding it to the pipeline now and make it build the same way our main image?

Comment (schoren) [reply agent/Dockerfile:1]: yea, totally, no time like the present

**Review by danielbdias — APPROVED** [review]

File: agent/Dockerfile (hunk 1 above)
Comment (mathnogueira) [reply agent/Dockerfile]: Done!

**Pull request merged**